### PR TITLE
feat: Add menu to download web app

### PR DESCRIPTION
**Pull request by karasusan**
## What
Added menu items to download web app.
RenderStreamingMenu

## Menu items 
```
Edit > Render Streaming > Download web app 
Edit > Render Streaming > Show web app documentation
Edit > Render Streaming > Show web app source code
```

### CODE REVIEW

**Comment by karasusan** [comment]
We need to recreate `unitypackage` includes this script.

File: Assets/Scripts/Editor/RenderStreamingMenu.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
+﻿using UnityEditor;
+using UnityEngine;
+using System.Net;
+
+namespace Unity.RenderStreaming.Editor
+{
+    public static class RenderStreamingMenu
+    {
+        const string PackageVersion = "1.0.0-preview";
```
[the remark this answers is not in the document]
Comment (karasusan) [reply Assets/Scripts/Editor/RenderStreamingMenu.cs]: yeah, we should do it but `RequestJobManager` is in `com.unity.renderstreaming` package.
I have no idea how to reference it 😞

[the remark this answers is not in the document]
Comment (sindharta) [reply Assets/Scripts/Editor/RenderStreamingMenu.cs]: Just use 
`using Unity.RenderStreaming.Editor`  ?

[the remark this answers is not in the document]
Comment (karasusan) [reply Assets/Scripts/Editor/RenderStreamingMenu.cs]: I changed dependencies of template package here. https://github.com/Unity-Technologies/UnityRenderStreaming/pull/131

**Pull request closed without merging**